### PR TITLE
update po file

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@codiebeulaine Please review.


Cleanup templates with translation. 
General rw and en po files

File: .gitignore [hunk 1]
```diff
@@ -61,3 +61,6 @@ gem_test.db
 gem_test.db
 node_modules/
 db.sqlite3-journal
+
+molo_makemessages.py
```
Comment (codiebeulaine) [inline .gitignore]: @Mitso what are these two files? Should they be in the gitignore for everyone?

File: gem/templates/rwanda/core/main.html [hunk 2]
```diff
@@ -5,23 +5,14 @@
     {% with cewekerenHost='cewekeren.com' ninyampingaHost='ninyampinga.com' qaHost='ninyampinga-rwanda.qa-hub.ie.gehosting.org'  %}
       {% if cewekerenHost in request.get_host %}
         <div class="freebasics-notification">
-          <p class="freebasics-notification__text">
-            {% blocktrans %}
-              Hei cewek keren! Apa kamu pelanggan Indosat? Kalo iya, kamu bisa mengakses
-              Girl Effect secara gratis lewat" %} <a href="http://www.freebasics.com" target="_top" class="freebasics-notification__anchor">Internet.org</a>
-              hanya dengan meng-klik pilihan "Girl Effect" yang tersedia di dalam menu. Keren banget, kan?
-            {% endblocktrans %}
+          <p class="freebasics-notification__text">Hei cewek keren! Apa kamu pelanggan Indosat? Kalo iya, kamu bisa mengakses Girl Effect secara gratis lewat<a href="http://www.freebasics.com" target="_blank" class="freebasics-notification__anchor">Internet.org</a> hanya dengan meng-klik pilihan "Girl Effect" yang tersedia di dalam menu. Keren banget, kan?
```
Comment (codiebeulaine) [inline gem/templates/rwanda/core/main.html]: should we have cerek keren in this one since these templates are just for RW?

Comment (Mitso) [reply gem/templates/rwanda/core/main.html]: Updated...

**Pull request merged**